core/src/com/unciv/models/ruleset/unique/Unique.kt

```diff
@@ -71,7 +71,8 @@ class Unique(val text: String, val sourceObjectType: UniqueTarget? = null, val s
 
     private fun conditionalApplies(
         condition: Unique,
-        state: StateForConditionals
+        state: StateForConditionals,
+        baseUnique: Unique
```

Comment:
This parameter feels redundant; it is a private function so only accessible from within this class, and is only ever accessed from `conditionalsApply`, where the value `this` is passed, which can also just be called in this function

Reply:
Good point :)